packages/apps/spaces/pages/auth/signin.tsx

```diff
@@ -36,7 +39,7 @@ export default function SignIn({
         <GridItem h='100vh'>
           <Box height='50%'>
             <Image
-              alt=''
+              alt='dotted background'
```

Comment:
Alt should be empty string if it's not providing useful information. Eg is user is represented by Avatar there should be alt filled with user name so person using assistive technologies can use the app. In this case bg has only estetic function and describing it will only create informational noice for users that rely on assistive technologies

Reply:
ok